gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -210,23 +245,23 @@ def __init__(self, controller, config):
                                                       self.config)
 
         self.status_map = {
-                'INIT': 'Created Network Service Device with status INIT.',
-                'PENDING_CREATE': '',
-                'DEVICE_SPAWNING': ('Creating NSD, launched the new device, ' +
-                                    'polling on its status'),
-                'DEVICE_UP': 'Device is UP/ACTIVE',
-                'HEALTH_CHECK_PENDING': ('Device health check is going on ' +
-                                        ' through configurator'),
-                'HEALTH_CHECK_COMPLETED': 'Health check succesfull for device',
-                'INTERFACES_PLUGGED': 'Interfaces Plugging successfull',
-                'PENDING_CONFIGURATION_CREATE': ('Started configuring device '
-                                                 + 'for routes, license, etc'),
-                'DEVICE_READY': 'Device is ready to use',
-                'ACTIVE': 'Device is Active.',
-                'DEVICE_NOT_UP': 'Device not became UP/ACTIVE',
+            'INIT': 'Created Network Service Device with status INIT.',
```

Comment:
**As we are changing the workflow, Can we add workflow in both NSO, NDO modules**